indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -56,7 +56,10 @@
                             {% for item in dynamic_columns %}
                                 <th class="i-table">{{ item.title }}</th>
                             {% endfor %}
-                            <th class="i-table thin-column"></th>
+                            {% if reviewed_abstracts %}
```

Comment:
Consistency. Why would reviewing something suddenly cause a column to appear in that list?